### ISSUE TITLE
 Noto Serif Gurmukhi is not attaching conjuncts correctly

### ISSUE THREAD

**Issue by simoncozens** [notofonts/gurmukhi#4]
Text: ਕ੍ਰਕ੍ਹਖ੍ਹ

![shape](https://user-images.githubusercontent.com/106728/175061812-d06c7854-edc2-43c6-ab69-b2ab565d3748.png)

Looks odd - is it meant to be attached?

**Comment by tiroj** [comment]
Note that if the subscripts do attach, there are different practices with regard to the ਕ as base. Some writers/typefaces connect the subscript from the descending stroke, but that usually requires a ligature to be done smoothly in a font. Other connect below the bowl, which is what I do in Tiro Gurmukhi, using an inserted connection stroke to bridge the gap between the bowl and the subscript:
<img width="170" alt="image" src="https://user-images.githubusercontent.com/5614500/175182311-0d1d29e6-468b-4df5-afa3-75d8a4749564.png">

**Comment by simoncozens** [comment]
Yes, looking again at the thicknesses and directions of those strokes it looks like they were never designed to connect. But I feel like we should at least be consistent - if kra is connected, then so should kha.
